test/unit/api-player-spec.js

```diff
@@ -33,7 +34,8 @@ describe('Peaks.player', function() {
 
     it('should return an updated time if it has been modified through the audio element', function(done) {
       p.on('player_seek', function(currentTime) {
-        var diff = Math.abs(p.player.getCurrentTime() - newTime);
+        // var diff = Math.abs(p.player.getCurrentTime() - newTime);
```

Comment:
This test is intended to check `player.getCurrentTime()`. We should add separate test to also check `currentTime` as reported in the `player_seek` event.

Reply:
ugh, sorry, I didn't notice this, it was just me testing..

Reply:
(p.s. reverted)